src/components/cards/TrendingPairs/TrendingPairs.vue

```diff
@@ -0,0 +1,71 @@
+<template>
```

Comment:
Convert to new component syntax

Reply:
yep, had a general comment on converting once functionality was complete

Reply:
Ahh sorry, missed that